### ISSUE TITLE
Can't use PS_ON pin because logic has no power

### ISSUE THREAD

**Issue by levlandau** [ultimachine/RAMBo#34]
I am trying to setup Rambo to turn on/ff the 24V power supply after the printing is done. [This guide](http://www.instructables.com/id/Converting-12v24v-power-supply-to-remote-G-code-sw/?ALLSTEPS) steps through how to do this with a relay.

I have Rambo rev 1.2h. It seems once the USB and logic 5V power were isolated, the logic no longer has power once the PSU is turned off. So it can't be used to turn the PSU on/off. The 5V pin next to the  PS_ON pin also has no power.

I have searched for documentation on how to resolve this problem, but there is nothing obvious. Rev 1.2h no longer has the jumper which allows one to select USB power for the logic side of the board.

Can someone explain how the PSU can be turned on/off correctly? Do I need to supply external 5V power as is suggested by [issue #33](https://github.com/ultimachine/RAMBo/issues/33) ??

**Comment by tonokip** [comment]
Yes, you need to power logic independently in order to control the power supply.

The solution of using external 5V power and a diode seems like a good one. 

Next to the bed fuse are 2 unpopulated pins labeled SMPS which you can use to disable the onboard 5V power supply. You can do this by soldering the two points together. This would allow you to use only the 5V external power to power the logic. 

Other then that, the process of controlling the PSU is the same.

**Comment by Tailslide** [comment]
Can someone please explain how to do this in more detail?  The 5V standby power goes to the pin closest to the power supply input and the PS_ON goes to the other pin?  Where does the diode go and which way?
